policy/Object.yaml

```diff
@@ -0,0 +1,392 @@
+    - base_PixelFlags_flag_clipped
+    - base_PixelFlags_flag_clippedCenter
+    - base_PixelFlags_flag_cr
+    - base_PixelFlags_flag_crCenter
+    - base_PixelFlags_flag_edge
+    - base_PixelFlags_flag_inexact_psf
+    - base_PixelFlags_flag_inexact_psfCenter
+    - base_PixelFlags_flag_interpolated
+    - base_PixelFlags_flag_interpolatedCenter
+    - base_PixelFlags_flag_offimage
+    - base_PixelFlags_flag_saturated
+    - base_PixelFlags_flag_saturatedCenter
+    - base_PixelFlags_flag_sensor_edge
+    - base_PixelFlags_flag_sensor_edgeCenter
+    - base_PixelFlags_flag_suspect
+    - base_PixelFlags_flag_suspectCenter
+    - base_ClassificationExtendedness_flag
+    - base_InputCount_flag
+    - base_InputCount_flag_noInputs
+    - calib_astrometry_used
+    - calib_photometry_reserved
+    - calib_photometry_used
+    - calib_psf_candidate
+    - calib_psf_reserved
+    - calib_psf_used
+    - slot_ApFlux_flag
+    - slot_ApFlux_flag_apertureTruncated
+    - slot_ApFlux_flag_sincCoeffsTruncated
+    - slot_ApFlux_instFlux
+    - slot_ApFlux_instFluxErr
+    - slot_CalibFlux_flag
+    - slot_CalibFlux_flag_apertureTruncated
+    - slot_CalibFlux_flag_sincCoeffsTruncated
+    - slot_CalibFlux_instFlux
+    - slot_CalibFlux_instFluxErr
+    - slot_Centroid_flag_almostNoSecondDerivative
+    - slot_Centroid_flag_edge
+    - slot_Centroid_flag_noSecondDerivative
+    - slot_Centroid_flag_notAtMaximum
+    - slot_Centroid_flag_resetToPeak
+    - slot_ShapeRound_flag
+    - slot_ShapeRound_flag_no_pixels
+    - slot_ShapeRound_flag_not_contained
+    - slot_ShapeRound_flag_parent_source
+    - slot_ShapeRound_Flux
+    - slot_ShapeRound_x
+    - slot_ShapeRound_xx
+    - slot_ShapeRound_xy
+    - slot_ShapeRound_y
+    - slot_ShapeRound_yy
+    # - forced source
+    - slot_PsfFlux_apCorr
+    - slot_PsfFlux_apCorrErr
+    - slot_PsfFlux_flag
+    - slot_PsfFlux_flag_apCorr
+    - slot_PsfFlux_flag_edge
+    - slot_PsfFlux_flag_noGoodPixels
+    - ext_photometryKron_KronFlux_apCorr
+    - ext_photometryKron_KronFlux_apCorrErr
+    - ext_photometryKron_KronFlux_flag
+    - ext_photometryKron_KronFlux_flag_apCorr
+    - ext_photometryKron_KronFlux_flag_bad_radius
+    - ext_photometryKron_KronFlux_flag_bad_shape
+    - ext_photometryKron_KronFlux_flag_bad_shape_no_psf
+    - ext_photometryKron_KronFlux_flag_edge
+    - ext_photometryKron_KronFlux_flag_no_fallback_radius
+    - ext_photometryKron_KronFlux_flag_no_minimum_radius
+    - ext_photometryKron_KronFlux_flag_small_radius
+    - ext_photometryKron_KronFlux_flag_used_minimum_radius
+    - ext_photometryKron_KronFlux_flag_used_psf_radius
+    - slot_PsfShape_flag
+    - slot_PsfShape_flag_no_pixels
+    - slot_PsfShape_flag_not_contained
+    - slot_PsfShape_flag_parent_source
+    - slot_Shape_flag
+    - slot_Shape_flag_no_pixels
+    - slot_Shape_flag_not_contained
+    - slot_Shape_flag_parent_source
+# Need to add per object Galactic extinction to post-processing
```

Comment:
Sims is now installed with the shared stack, so could this be added using functions there (as is done in `pipe_analysis`). It’s not part of `lsst_distrib`, but then neither is `qa_explorer`, yet... I’m not sure where the dependency boundaries lie...